### PR TITLE
desgin-lecturer-view

### PR DESCRIPTION
**Pull request by kienpham9819**
## Related Tickets:
- [#30939](https://edu-redmine.sun-asterisk.vn/issues/30939)

## WHAT this PR does?
Design lecturer view

## Notes Impacted Areas:
- Screencast from 22-10-2020 13_49_55

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kienpham9819** [comment]
ready

File: app/Http/Controllers/LecturerController.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+class LecturerController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function listCourse()
+    {
+        $courses = auth()->user()->teaches()->paginate(10);
```
Comment (huyenttm-0901) [inline app/Http/Controllers/LecturerController.php]: đưa số vào config

File: routes/web.php [hunk 2]
```diff
@@ -39,3 +39,13 @@
...
+
+Route::get('course-list-student', 'StudentController@listCourse')->name('students.courseList');
+Route::get('course-detail-student/{course}', 'StudentController@showDetailCourse')->name('students.courseDetail');
+Route::get('group-detail-student/{group}', 'StudentController@showDetailGroup')->name('students.groupDetail');
+
+Route::get('course-list-lecturer', 'LecturerController@listCourse')->name('lecturers.courseList');
+Route::get('course-detail-lecturer/{course}', 'LecturerController@showDetailCourse')->name('lecturers.courseDetail');
+Route::get('showFormEditGroup/{group}', 'LecturerController@showFormEditGroup')->name('lecturers.showFormEditGroup');
+Route::patch('updateGroup/{group}', 'LecturerController@updateGroup')->name('lecturers.updateGroup');
+Route::get('groupDetail/{group}', 'LecturerController@groupDetail')->name('lecturers.groupDetail');
```
Comment (huyenttm-0901) [inline routes/web.php]: xem lại url

**Comment by kienpham9819** [comment]
ready

**Pull request merged**